### ISSUE TITLE
Missing Meshes ?

### ISSUE THREAD

**Issue by ADz1075** [TitansBane/Elysium#115]
Hi, 

I have just installed the full download for Elysium using Wabbajack. I followed all instructions and all steps, everything is running smoothly and I am so very grateful for you for taking the chore out of modding Skyrim.

I have one minor issue; some Draugr (but not all) seem to have missing meshes; I've taken a screenshot with details (attached) 
![drauger missing mesh](https://user-images.githubusercontent.com/87277561/125204790-defbe300-e276-11eb-984d-08207f919c87.png) 

I checked for open and closed issues and couldn't see anything pertaining to this. 

I have tried a reinstall from within mod manager for anything with 'draugr' in the name and also SMIM - but no luck so far getting the missing Draugr to appear.

Any help gratefully received! :)

Cheers, 

ADz

**Comment by TitansBane** [comment]
Already fix this issue in a previous update. You'll need to redownload the list again and update it through Wabbajack.

**Comment by ADz1075** [comment]
Hi 

I did check the changelog; and saw the note on the 29th of June about Draugr but didn't see anything after that point (I downloaded on the 7th of July, so wrongly 'assumed' I didn't need to check for updates again) sorry for the assumption and for not checking the obvious update route first before posting. 

Thanks for getting back to me so quickly! And thanks for all your hard work making and keeping this up-to-date. You are a god among men! 

ADz.
